apps/src/gamelab/spritelab/worldCommands.js

```diff
@@ -20,9 +20,11 @@ export const commands = {
   },
 
   setBackgroundImage(img) {
-    let backgroundImage = this.loadImage(img);
-    backgroundImage.name = img;
-    spriteUtils.background = backgroundImage;
+    if (this._predefinedBackgrounds[img]) {
```

Comment:
Two questions:
1. Should there be an else here?
2. Do we use loadImage anywhere anymore? If now, let's remove it :)

Reply:
1. No, I think we decided yesterday not to try to live-load the image if we don't have it preloaded. That case should never come up, so it would basically just be untested/unused code. The image parameter can only be set by the values in the dropdown, so as long as we have all of those images preloaded, we will never reach an else case.
2. `loadImage` is only called during the preload stage now.

Reply:
👍